src/webui/src/App.tsx

```diff
@@ -27,15 +27,15 @@ class App extends React.Component<{}, AppState> {
         };
     }
 
-    async componentDidMount() {
+    async componentDidMount(): Promise<any> {
```

Comment:
This should be Promise<void> since there is no return value.

Reply:
Thanks. Had changed `Promise<any>` to `Promise<void>`.